### ISSUE TITLE
Cancelling certain queued command takes 2 tries

### ISSUE THREAD

**Issue by Jazcash** [beyond-all-reason/Beyond-All-Reason#926]
Usually, to cancel a command in a queue, you simply need to issue that same command again and it will be removed. However, once shift is released, it requires issuing the command twice instead of once. Doesn't seem to be a problem with all commands, but guard is definitely one them, so coming back to a unit at some point in the game to cancel its queued guard is awkward. 

![image](https://user-images.githubusercontent.com/1434248/162620157-523688a8-3664-4391-b8f7-db7789752beb.png)

**Comment by WatchTheFort** [comment]
Could this be caused by the https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/luaui/Widgets/cmd_guard_remove.lua widget?
It sounds like:
1. Widget removes old guard command
2. Engine processes new guard command
3. Since old command was removed, new command ends up adding guard order

**Comment by Jazcash** [comment]
> Could this be caused by the [`cmd_guard_remove`](https://github.com/beyond-all-reason/Beyond-All-Reason/blob/master/luaui/Widgets/cmd_guard_remove.lua) widget? It sounds like:
> 
> 1. Widget removes old guard command
> 2. Engine processes new guard command
> 3. Since old command was removed, new command ends up adding guard order

Yup, the problem seems to be caused by this widget indeed, turning it off fixes the issue
